experimental/segmenter/tools/generate_properties.py

```diff
@@ -73,6 +104,12 @@
                     # codepoints 0x0029 and 0x005D with CP line break class both
                     # have EA property "Na".
                     lb_props[i] = "CP_EA"
+                elif uni_prop == "Cn":
+                    # For LB30b
+                    if prop == "ID" and extended_pictographics[i] == "Y":
```

Comment:
Nit: Maybe quote the spec here "Unassigned codepoints with Line_Break=ID in some blocks are also assigned the Extended_Pictographic property." 

http://www.unicode.org/reports/tr51/#def_level1_emoji